### ISSUE TITLE
How to calculate CP/CR/CF1/OP/OR/OF1 of top-3?

### ISSUE THREAD

**Issue by Rmuli** [sacmehta/EdgeNets#20]
![top3](https://user-images.githubusercontent.com/37521721/65810576-6bdac700-e1de-11e9-8fc6-52e55883f4a4.png)

**If I want to calculate CP/CR/CF1/OP/OR/OF1 of top-3, is it correct for me to add the following code in the original program?**

`

    no_examples = target.shape[0] # 获得样本数 ——> batch_size的大小
    output=torch.sigmoid(output)  #  sigmoid激活
    output=output.cpu().detach().numpy()
    top3 = np.zeros_like(output) 
    for ind_example in range(no_examples):
        top_pred_inds = np.argsort(output[ind_example])[::-1] # 概率排序
        for k in range(3):
            top3[ind_example, top_pred_inds[k]] = 1 # 概率前3的置为1, 其余为0
    pred=torch.from_numpy(top3).long().cuda() #  numpy——>tensor 与下文保持一致
    print("pred: ", pred)
    print("target: ", target)
`

**Comment by sacmehta** [comment]
I don't think that we reported this table in any of our work. Please ask the authors of the corresponding paper.
